index.js

```diff
@@ -111,17 +112,51 @@ class PkgPyFuncs {
       throw new this.serverless.classes.Error(`[serverless-package-python-functions] ${ret.error.message}`)
     }
 
+    const out = ret.stdout.toString()
+
     if (ret.stderr.length != 0){
-      this.log(ret.stderr.toString())
+      const errorText = ret.stderr.toString().trim()
+      const countErrorNewLines = errorText.split('\n').length
+
+      if(countErrorNewLines < 2 && errorText.toLowerCase().includes('git clone')){
+        // Ignore false positive due to pip git clone printing to stderr
+        console.log(errorText)
+      } else if(errorText.toLowerCase().includes('docker')){
+        console.log('stdout:', out)
+        console.log('stderr: ', errorText)
+        this.error("Docker Error Detected")
+
+      } else {
+        // Error is not false positive, 
+        console.log('___ERROR DETECTED, BEGIN STDOUT____\n', out)
+        console.log('---BEGIN STDERR---\n', errorText)
+        this.requestUserConfirmation()
+      }
+
     }
 
-    const out = ret.stdout.toString()
     return out
   }
 
+  requestUserConfirmation(prompt="\n\n??? Do you wish to continue deployment with the stated errors? \n",
```

Comment:
Let's put the `abort on error` feature behind a configuration flag. This change may break the config of someone who is cool with the errors